.prettier.rs

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Rename file in `.prettierrc` and after that rerun prettier:fix script

Reply:
Done!